### PR TITLE
Update batcher and proposer for sepolia-alpha

### PR DESCRIPTION
**Pull request by cody-wang-cb**
A script prepared to update batcher and proposer to keychain key (0x7A43fD33e42054C965eE7175dd4590D2BDba79cB) in sepolia-alpha.

Overall migration runbook: https://docs.google.com/document/d/1pP2apZYK7kyuKy-e_T9Y-guu_css-JL48xQIe-USkqQ/edit

Tested all the contracts locally via `vm.prank()`

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>1</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>2</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: sepolia-alpha/2024-08-21-update-batcher-proposer/.env [hunk 1]
```diff
@@ -0,0 +1,17 @@
+OP_COMMIT=e3ba24e72085d85bb5584dda33a03ccf60db86f0
+BASE_CONTRACTS_COMMIT=e961e8638874d571aa63a6e35507e3779131d1eb
+
+# Batcher
+L1_SYSTEM_CONFIG_ADDRESS=0x7F67DC4959cb3E532B10A99F41bDD906C46FdFdE
+SYSTEM_CONFIG_OWNER=0xAf6E0E871f38c7B653700F7CbAEDafaa2784D430
+
+NEW_BATCH_SENDER=0x0000000000000000000000007A43fD33e42054C965eE7175dd4590D2BDba79cB
+ROLLBACK_BATCH_SENDER=0x000000000000000000000000212dd524932bc43478688f91045f2682913ad8ee
```
Comment (mdehoog) [inline sepolia-alpha/2024-08-21-update-batcher-proposer/.env:9]: nit: might be good to use addresses here and convert them to the byte32 in the script itself?

File: sepolia-alpha/2024-08-21-update-batcher-proposer/script/UpdateProposer.sol [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import "forge-std/Script.sol";
+
+contract UpdateProposer is Script {
+    address internal NEW_PROPOSER = vm.envAddress("NEW_PROPOSER");
+    address internal L2_OUTPUT_ORACLE_PROXY = vm.envAddress("L2_OUTPUT_ORACLE_PROXY");
+    address internal PROXY_ADMIN = vm.envAddress("L1_PROXY_ADMIN");
+    address internal PROXY_ADMIN_OWNER = vm.envAddress("PROXY_ADMIN_OWNER");
+
+    function run() public {
+        L2OutputOracle l2OOProxy = L2OutputOracle(L2_OUTPUT_ORACLE_PROXY);
```
Comment (mdehoog) [inline sepolia-alpha/2024-08-21-update-batcher-proposer/script/UpdateProposer.sol:15]: should this be https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/dispute/DisputeGameFactory.sol, now that we have fault proofs on sepolia-alpha?

Comment (cody-wang-cb) [reply sepolia-alpha/2024-08-21-update-batcher-proposer/script/UpdateProposer.sol:15]: Ahh I see it now `OP_PROPOSER_GAME_FACTORY_ADDRESS`. Let me redo this

**Pull request merged**